core/src/main/java/org/n52/iceland/request/AbstractServiceRequest.java

```diff
@@ -175,6 +180,18 @@ public Locale getRequestedLocale() {
         return LocaleHelper.fromString(getRequestedLanguage());
     }
 
+    public String getOriginalRequest() {
```

Comment:
Please talk about it in the Huddle - I am OK with exposing the optional for all new fields. As a step further, we could even expose the optional for requestedLanguage and mark the ``isSet...()`` as deprecated.